src/utils.ts

```diff
@@ -2,9 +2,7 @@ import { readdirSync } from 'fs'
 import { inspect } from 'util'
 import { groupBy, uniq } from 'lodash'
 
-const HOLOCHAIN_DATABASE_DIRECTORY = process.env.NODE_ENV === 'test'
-  ? './'
-  : '/var/lib/holochain-rsm/databases_lmdb4'
+const HOLOCHAIN_DATABASE_DIRECTORY = process.env.HOLOCHAIN_DATABASE_DIRECTORY || './'
```

Comment:
Can we add a check that throws an error or at least a warning whenever we're in production mode and `process.env.HOLOCHAIN_DATABASE_DIRECTORY` is not defined?

Reply:
done